apps/date-component/README.md

```diff
@@ -0,0 +1,63 @@
+# Date component example
+
+This tutorial explains the steps to create a date using the [hof-component-date](https://github.com/UKHomeOfficeForms/hof-component-date)
+
+## Getting started
+
+Create a hof app called date-component 
```

Comment:
Remember that these are tutorials for a user to add these pieces of functionality to *their own apps*, not just to recreate the example.

Code examples are useful, but they're not the goal here.